source/features/table-input.tsx

```diff
@@ -0,0 +1,80 @@
+import './table-input.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+import * as textFieldEdit from 'text-field-edit';
+
+import features from '.';
+import smartBlockWrap from '../helpers/smart-block-wrap';
+
+function generateHtmlTable(width: number, height: number): string {
+	return '<table>\n' + ('<tr>\n' + '\t<td>\n'.repeat(width)).repeat(height) + '</table>\n';
+}
+
+function addTable(event: delegate.Event<MouseEvent, HTMLButtonElement>): void {
+	const field = event.delegateTarget.form!.querySelector('textarea')!;
+	const cursorPosition = field.selectionStart;
+
+	textFieldEdit.insert(field, smartBlockWrap(generateHtmlTable(Number.parseInt(event.delegateTarget.dataset.x!, 10), Number.parseInt(event.delegateTarget.dataset.y!, 10)), field));
+
+	field.focus();
+	const firstRowPosition = field.value.indexOf('<td>', cursorPosition) + '<td>'.length;
+	field.setSelectionRange(firstRowPosition, firstRowPosition);
+}
+
+function highlightSquares({delegateTarget: hover}: delegate.Event<MouseEvent, HTMLElement>): void {
+	for (const cell of Array.from(hover.parentElement!.children) as HTMLElement[]) {
+		cell.classList.toggle('selected', cell.dataset.x! <= hover.dataset.x! && cell.dataset.y! <= hover.dataset.y!);
+	}
+}
+
+function init(): void {
+	delegate(document, '.rgh-table-input-cell', 'click', addTable);
+	delegate(document, '.rgh-table-input-cell', 'mouseover', highlightSquares);
+
+	for (const anchor of select.all('md-task-list')) {
+		anchor.after(
+			<details className="details-reset details-overlay flex-auto toolbar-item select-menu select-menu-modal-right hx_rsm">
+				<summary
+					className="text-center menu-target py-2 p-md-1 hx_rsm-trigger ml-1"
+					role="button"
+					aria-label="Add a table"
+					aria-haspopup="menu"
+				>
+					{/* eslint-disable react/no-unknown-property */}
```

Comment:
The error specifically tells you what to do, don't silence it

```
  ✖  48:713  Unknown property clip-rule found, use clipRule instead        react/no-unknown-property
  ✖  48:733  Unknown property fill-rule found, use fillRule instead        react/no-unknown-property
  ✖  49:64   Unknown property stroke-width found, use strokeWidth instead  react/no-unknown-property
  ✖  50:66   Unknown property stroke-width found, use strokeWidth instead  react/no-unknown-property
```

Reply:
I silenced it because `strokeWidth` and al. weren't converted in their snake-case equivalents during build. Is there another way to circumvent this?

Reply:
Indeed, that's a dom-chef bug. Let's mention it:
```suggestion
					{/* eslint-disable react/no-unknown-property -- TODO: until https://github.com/vadimdemedes/dom-chef/issues/72*/}
```